### DISCUSSION TITLE
现在的swoole还有很多bug码

### DISCUSSION THREAD

**Discussion by a6965921** [swoole/swoole-src#4596] [Q&A]
现在的swoole还有很多bug码    我感觉不精通C语言 没办法用了。。

**Comment by Yurunsoft** [comment]
如果你发现了swoole的bug，可以反馈。张口就来bug多，不知道依据是什么呢？
如果你没有发现swoole的bug，哪来的这个问题？
另外纠正错别字：“码”=>“吗”

**Reply by sy-records** [reply]
🤣 看起来他这个字错的很久了。

![image](https://user-images.githubusercontent.com/33931153/152666741-90c5b08d-cfa3-4832-b6ec-95beeca1a05e.png)
